ui/app/components/search-select.js

```diff
@@ -0,0 +1,99 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  'data-test-component': 'search-select',
+  classNames: ['field', 'search-select'],
+  store: service(),
+  /*
+   * @public
+   * @param Function
+   *
+   * Function called when any of the inputs change
+   * accepts a single param `value` that is the
+   * result of calling `toVal()`.
+   *
+   */
+  onChange: () => {},
+
+  /*
+   * @public
+   * @param String | Array
+   * A comma-separated string or an array of strings.
+   * Defaults to an empty array.
+   *
+   */
+  inputValue: computed(function() {
+    return [];
+  }),
+  selectedOption: null,
+  selectedOptions: null,
+  options: null,
```

Comment:
What are the differences between these three - can we add some comments to clarify usage of the component?